packages/api/src/graphql/subscription.ts

```diff
@@ -50,7 +50,7 @@ export const GraphQLSubscription = new GraphQLObjectType<Subscription, Context>(
     createdAt: {type: GraphQLNonNull(GraphQLDateTime)},
     modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
     user: {
-      type: GraphQLNonNull(GraphQLUser),
+      type: GraphQLUser,
```

Comment:
in which case does a subscription not have an user?

Reply:
If user gets deleted (possible in the editor).